hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/cache/CachePutAllMessageTask.java

```diff
@@ -70,14 +75,24 @@ public String getDistributedObjectName() {
 
     @Override
     public Object[] getParameters() {
+        Map<Data, Data> map = createMap();
```

Comment:
This is to make the interceptor test at [here](https://github.com/hazelcast/hazelcast-enterprise/blob/master/hazelcast-enterprise/src/test/java/com/hazelcast/client/security/CacheSecurityInterceptorTest.java#L105) work properly and it makes sense since the parameter of the `ICache.putAll` method receives a map. Yes, it is one more map creation but it makes the API more user friendly. Let me know if it is OK or I am happy to revert it if we agree on returning the `parameters.entries`
```     
public java.util.List<java.util.Map.Entry<com.hazelcast.internal.serialization.Data, com.hazelcast.internal.serialization.Data>> entries;
```